Lesson 5/proga2.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import "fmt"
+
+// Есть массив символов.
+// Собрать из них строку, но в нее добавлять только символы латинского алфавита, пробелы и точки.
+// Вывести результирующую строку.
+
+func main() {
+	someRandomSymbol := [11]rune{'y', 'w', 'W', ',', '.', '1', '3', '1', 'f', 'S', ' '}
+	var newRandomSymbol []string
+	var name string
+	for i, _ := range someRandomSymbol {
+		n := int(someRandomSymbol[i])
+		name = string(n)
+		if ((n < 90) && (n > 65)) || ((n < 122) && (n > 95)) || (n == 32) || (n == 46) {
```

Comment:
```suggestion
                n := rune(someRandomSymbol[i])
		if ((n < 90) && (n > 65)) || ((n < 122) && (n > 95)) || (n == 32) || (n == 46) {
```
name = string(n)
if ((n > 'A') && (n < 'Z')) || ((n > 'a') && (n < 'z')) || (n == '.') || (n == ' ') {

Можно сравнивать с символами на прямую. Таким образом исключим классическую ошибку с ошибкой в порядке символа в аски таблице.